src/main/java/dev/jbang/dependencies/ArtifactResolver.java

```diff
@@ -354,13 +357,13 @@ private Settings newEffectiveSettings() {
 			// find the settings
 			Path settingsFile = settingsXml;
 			if (settingsFile == null) {
-				Path userSettings = Paths.get(System.getProperty("user.home"), ".m2", "settings.xml");
-				if (Files.exists(userSettings)) {
-					settingsFile = userSettings.toAbsolutePath();
+				Path globalSettings = Paths.get(System.getProperty("user.home"), ".m2", "settings.xml");
+				if (Files.exists(globalSettings)) {
+					settingsFile = globalSettings.toAbsolutePath();
 				}
 			}
 			if (settingsFile != null) {
-				settingsBuilderRequest.setUserSettingsFile(settingsFile.toFile());
+				settingsBuilderRequest.setGlobalSettingsFile(settingsFile.toFile());
```

Comment:
What exactly is the difference between using global and user?

Reply:
In our specific currrent situation probably not much but thought might as well just treat home dir settings as global settings if we ever want to introduce using user settings That can override global settings.